### PR TITLE
sig-k8s-infra: Default prowjobs to k8s-infra

### PR DESCRIPTION
**Pull request by ameukam**
Ensure prowjobs will run in the community infrastructure

Signed-off-by: Arnaud Meukam <ameukam@gmail.com>

### CODE REVIEW

**Review by neolit123 — COMMENTED** [review]
/lgtm
/approve

thanks @ameukam
you have to also pr this change in k/test-infra

File: kinder/ci/tools/update-workflows/templates/testinfra/kubeadm-kinder-dryrun.yaml [hunk 1]
```diff
@@ -1,4 +1,5 @@
-- name: ci-kubernetes-e2e-kubeadm-kinder-dryrun-{{ dashVer .KubernetesVersion }}
+- name: ci-kubernetes-e2e-kubeadm-kinder-unversioned-kubelet-config-map-{{ dashVer .KubernetesVersion }}
```
Comment (neolit123) [inline kinder/ci/tools/update-workflows/templates/testinfra/kubeadm-kinder-dryrun.yaml:1]: /hold
is this a mistake or the diff is confusing?

Comment (ameukam) [reply kinder/ci/tools/update-workflows/templates/testinfra/kubeadm-kinder-dryrun.yaml]: It was a mistake. Fixed.

**Pull request merged**